src/main/java/lotto/domain/LottoPaper.java

```diff
@@ -38,6 +47,12 @@ Integer getMatchCount(List<Integer> winningNumber) {
                 .count();
     }
 
+    public boolean isContain(int number) {
+        return Optional.ofNullable(number)
+                .map(lottoNumbers::equals)
+                .isPresent();
+    }
+
```

Comment:
다음과 같이 변경할 수 있을 것 같아요 :)
```suggestion
    public boolean contain(LottoNum bonusBall) {
        return lottoNumbers.contains(bonusBall);
    }

```